sw/device/lib/crypto/drivers/hmac.c

```diff
@@ -397,17 +480,21 @@ status_t hmac_update(hmac_ctx_t *ctx, const uint8_t *data, size_t len) {
   // Keep writing incoming bytes
   msg_fifo_write(data, len - leftover_len);
 
+  // delay here, even  this small is enough
+  for (size_t i = 0; i < 40; i = launder32(i + 1))
+    ;
```

Comment:
We probably shouldn't merge this code; my understanding is that it's just here to force the bug to trigger so that we can see if the change works, but it shouldn't be in the final PR, right?